src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,100 @@
+.main {
+  @extend %flex-center;
+}
+
+.page__main {
+  margin-top: 50px;
+}
+
+.catalog-wrapper {
+  @extend %flex-center;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  max-width: $wrapper-width;
+
+  &__card {
+    margin: 0 $margin-between-cards 50px;
+  }
```

Comment:
let's use flex instead of setting each card a margin
hint: utilise justify-content property on the parent container (but not center)